constructor/nsis/OptionsDialog.nsh

```diff
@@ -86,13 +86,28 @@ Function mui_AnaCustomOptions_Show
         ${NSD_CreateCheckbox} 0 "$5u" 100% 11u "Add ${NAME} to $1 &PATH environment variable"
         IntOp $5 $5 + 11
         Pop $mui_AnaCustomOptions.AddToPath
+
+        # To address CVE-2022-26526.
+        # In AllUsers install mode, disable AddToPath as an option.
+        ${If} $InstMode = ${ALL_USERS}
```

Comment:
What if we don't show this option at all (and disable it?). All we have to do is make this condition wrap the block below. We should also update the docs in `construct.py` for the relevant keys.

Reply:
Thanks @jaimergp. Implemented your suggestion and built a test installer. Here's what happens:

Select "JustMe" installation.
![Screen Shot 2022-12-06 at 7 55 26 PM](https://user-images.githubusercontent.com/9688260/206063111-01585980-aa50-4fa1-a1e1-3f2724fc5aba.png)

The user sees the option to "AddToPath".
![Screen Shot 2022-12-06 at 7 57 07 PM](https://user-images.githubusercontent.com/9688260/206063113-e764c8bf-ce0e-4e57-940d-764a941e2d7a.png)

Select "AllUsers" installation.
![Screen Shot 2022-12-06 at 7 57 14 PM](https://user-images.githubusercontent.com/9688260/206063159-3bd51f26-23ec-477f-89c7-a396fbe4e6ff.png)

The option to "AddToPath" does not show up at all.
![Screen Shot 2022-12-06 at 7 59 29 PM](https://user-images.githubusercontent.com/9688260/206063233-68432896-7dad-413b-988b-64704594ffad.png)

I also tested this when running from the CLI and see the same results.

Reply:
This looks very good to me, great work!